src/vtp/cast_ballot.py

```diff
@@ -19,316 +19,36 @@
 
 """cast_ballot.py - command line level test script to automatically cast a ballot.
 
-See './cast_ballot.py -h' for usage information.
-
-See ../../docs/tech/executable-overview.md for the context in which this file was created.
-
+See 'cast_ballot.py -h' for usage information.
 """
 
 # pylint: disable=wrong-import-position
-import argparse
-import logging
-import os
-import pprint
-import random
 import sys
-import traceback
 
-import pyinputplus
-
-# Local imports
-from vtp.utils.address import Address
-from vtp.utils.ballot import Ballot, BlankBallot, Contests
-from vtp.utils.common import Globals, Shellout
+from vtp.script_libs.cast_ballot_lib import CastBallotLib
```

Comment:
Rename `script_libs` to `operations` or `ops`.

Reply:
Done - [a8977b0](https://github.com/TrustTheVote-Project/VoteTrackerPlus/pull/60/commits/a8977b04db61d8669631cf92980992f13377a77c)